spec/datatypes/datetime.md

```diff
@@ -0,0 +1,27 @@
+<!---
+SPDX-License-Identifier: Community-Spec-1.0
+SPDX-FileCopyrightText: 2022 Sebastian Crane <seabass-labrax@gmx.com>
+-->
+
+# DateTime
+
+`DateTime` is a signed integer of milliseconds since or before the Unix Epoch[^epoch].
+`DateTime` MUST be expressed as a series of base-10 digits, and MUST NOT be expressed with any leading zeroes, an exponent, a decimal point or a fractional part.
```

Comment:
this is just the default for all integer values here. And thus should be specified there?